ab_testing_tool/requirements/base.txt

```diff
@@ -1,6 +1,6 @@
 Django>=1.7
 ims_lti_py
-dj-static
+dj-static>=0.0.6
```

Comment:
Was there an issue with an older version of dj-static?  Just wondering as none of our other projects seem to specify a version limit.